src/pages/Projects.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+
+import '../styles/pages/Projects.css'
+
+import Diana from '../images/Diana.svg'
+
+import BackgroundLeftDetail from '../images/BackgroundBottomLeftDetailLight.svg'
+import BackgroundRightDetail from '../images/BackgroundBottomRightDetailLight.svg'
+
+
+import ProjectCard from '../components/ProjectCard'
+
+
+export default function Projects() {
+    var trends_props = {
+        name:"BR.TRENDS",
+        presentation_image:"https://cdn.theatlantic.com/thumbor/nTObmgixnAr5Gei2QRLcJ1Kl4R8=/438x0:1563x1125/500x500/media/img/mt/2020/05/Webart_TwitterTrump/original.jpg",
+        presentation_image_gradient_color:"",
+        description:"A tool to get Brazilian trending topics",
```

Comment:
Can we get these descriptions from Github? It could be done on another task, though.

Reply:
We'll, but on another PR. We have a task to integrate this page with Github (#6 specifically).